packages/ui-dom-utils/src/index.ts

```diff
@@ -48,6 +49,7 @@ export { isVisible } from './isVisible'
 export { ownerDocument } from './ownerDocument'
 export { ownerWindow } from './ownerWindow'
 export { requestAnimationFrame } from './requestAnimationFrame'
+export type { RequestAnimationFrameType } from './requestAnimationFrame'
 export { supportsObjectFit } from './supportsObjectFit'
```

Comment:
we can remove supportsObjectFit, its an IE hack https://caniuse.com/object-fit
not necessarly here, but if its not removed we should make a ticket to remove it.

Reply:
Made a ticket for it (INSTUI-3141)